lib/cachex.ex

```diff
@@ -529,10 +528,13 @@ defmodule Cachex do
       ...>   val2 = Cachex.get!(worker, "key2")
       ...>   [val1, val2]
       ...> end)
-      { :ok, [ "value1", "value2" ] }
+      [ "value1", "value2" ]
+
+      iex> Cachex.execute(:non_existant_cache, fn _worker -> nil end)
```

Comment:
We do not need this example addition as it's not specific to `execute/2` - it's common to all of the API.

Reply:
k, done!